### PR TITLE
Update sensor.ru.json

### PR DESCRIPTION
**Pull request by drts-sv**

### CODE REVIEW

File: custom_components/thermal_comfort/translations/sensor.ru.json [hunk 1]
```diff
@@ -7,14 +7,25 @@
...
+      "dry": "Слегка суховато",
       "very_comfortable": "Очень комфортно",
       "comfortable": "Комфортно",
-      "ok_but_humid": "Нормально, но сыровато",
-      "somewhat_uncomfortable": "Слегка некомфортно",
-      "quite_uncomfortable": "Очень влажно, некомфортно",
-      "extremely_uncomfortable": "Максимально влажно, угнетающе",
-      "severely_high": "Очень высокая влажность, опасно для астматиков"
+      "ok_but_humid": "Хорошо, но немного влажно",
+	  "somewhat_uncomfortable":"Немного некомфортно",
```
Comment (rautesamtr) [inline custom_components/thermal_comfort/translations/sensor.ru.json:14]: indentation

File: custom_components/thermal_comfort/translations/sensor.ru.json [hunk 2]
```diff
@@ -7,14 +7,25 @@
...
+      "quite_uncomfortable": "Очень влажно, довольно некомфортно",
+      "extremely_uncomfortable": "Максимально влажно, крайне некомфортно",
+      "severely_high": "Очень высокая влажность, опасно для больных астмой"
+    },
+	"thermal_comfort__simmer_zone": {
```
Comment (rautesamtr) [inline custom_components/thermal_comfort/translations/sensor.ru.json:19]: indentation

**Review by rautesamtr — CHANGES_REQUESTED** [review]
Thanks, just fix the indentation issues and this is good to go.

File: custom_components/thermal_comfort/translations/sensor.ru.json [hunk 3]
```diff
@@ -7,14 +7,25 @@
...
       "comfortable": "Комфортно",
-      "ok_but_humid": "Нормально, но сыровато",
-      "somewhat_uncomfortable": "Слегка некомфортно",
-      "quite_uncomfortable": "Очень влажно, некомфортно",
-      "extremely_uncomfortable": "Максимально влажно, угнетающе",
-      "severely_high": "Очень высокая влажность, опасно для астматиков"
+      "ok_but_humid": "Хорошо, но немного влажно",
+	  "somewhat_uncomfortable":"Немного некомфортно",
+      "quite_uncomfortable": "Очень влажно, довольно некомфортно",
+      "extremely_uncomfortable": "Максимально влажно, крайне некомфортно",
```
Comment (IATkachenko) [inline custom_components/thermal_comfort/translations/sensor.ru.json:16]: I'm not sure that we may discuss it in Russian...

Will it be better to use
```

      "quite_uncomfortable": "Слишком влажно, довольно некомфортно",
      "extremely_uncomfortable": "Очень влажно, крайне некомфортно",
```
?

"Максимально влажно" is not looks like native phrase...

**Pull request closed without merging**